README.md

```diff
@@ -1,5 +1,5 @@
 <!-- Update the title -->
-# Terraform Modules Template Project
+# IBM Secure Landing Zone VSI Autoscale Module
 
 <!--
 Update status and "latest release" badges:
```

Comment:
I suggest to update the "Incubating" badge to "Stable" (since it will be once this PR is merged). Use this:

```
[![Stable (With quality checks)](https://img.shields.io/badge/Status-Stable%20(With%20quality%20checks)-green)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
```

Reply:
Also the `latest release` badge below needs to be updated to point to this repo and not the module template